test/e2e/specs/editor/various/link-color-support.spec.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * WordPress dependencies
+ */
+import { test, expect } from '@wordpress/e2e-test-utils-playwright';
+
+test.describe( 'Link color in themes', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should apply selected link color in editor and frontend', async ( {
+		page,
+		editor,
+		pageUtils,
+	} ) => {
+		await editor.insertBlock( {
+			name: 'core/paragraph',
+		} );
+
+		await page.keyboard.type(
+			'This paragraph will have a custom link color set'
+		);
+
+		await page.keyboard.down( 'Shift' );
+		for ( let i = 0; i < 21; i++ ) {
+			await page.keyboard.press( 'ArrowLeft' );
+		}
+		await page.keyboard.up( 'Shift' );
+
+		await editor.clickBlockToolbarButton( 'Link' );
+
+		await page.keyboard.type( 'https://example.com' );
+		await pageUtils.pressKeys( 'Enter' );
+
+		await expect.poll( editor.getBlocks ).toMatchObject( [
+			{
+				name: 'core/paragraph',
+				attributes: {
+					content:
+						'This paragraph will have a <a href="https://example.com">custom link color set</a>',
+				},
+			},
+		] );
```

Comment:
```suggestion
		await editor.insertBlock( {
			name: 'core/paragraph',
			attributes: {
				content:
					'This paragraph will have a <a href="https://example.com">custom link color set</a>',
			},
		} );
```

This part has nothing to do with link color support and seems redundant to me - I think it covers enough [here](https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/specs/editor/blocks/links.spec.js) whether we can apply a link to selected text.